src/amo/components/AddonAdminLinks/index.js

```diff
@@ -79,19 +79,6 @@ export class AddonAdminLinksBase extends React.Component<InternalProps> {
       </li>
     ) : null;
 
-    const adminStatusLink = hasAdminPermission ? (
-      <li>
-        <a
-          className="AddonAdminLinks-admin-status-link"
-          href={`/admin/addon/manage/${addon.slug}/`}
-        >
-          {
-            // translators: This action allows an admin to edit the status of an add-on.
-            i18n.gettext('Admin add-on status')
-          }
-        </a>
-      </li>
-    ) : null;
 
```

Comment:
Seems like you need to remove the extra line as well, make sure to run `yarn prettier-dev`

Reply:
@championshuttler ok I will just remove it